_api-reference/search-apis/explain.md

```diff
@@ -12,11 +12,11 @@ redirect_from:
 **Introduced 1.0**
 {: .label .label-purple }
 
-Wondering why a specific document ranks higher (or lower) for a query? You can use the explain API for an explanation of how the relevance score (`_score`) is calculated for every result.
+Wondering why a specific document ranks higher (or lower) for a query? You can use the Explain API for an explanation of how the relevance score (`_score`) is calculated for every result.
 
```

Comment:
Line 15: Is "for a query" necessary here? Should "is calculated" be "was calculated"?